stdlib/experimental/geo/README.md

```diff
@@ -0,0 +1,201 @@
+## Package `geo`
+
+The package provides functions for geographic location filtering and grouping.
+It uses golang implementation of S2 Geometry Library [https://s2geometry.io/].
+It is designed to work on a schema with a tags `_ci` which contains S2 cell ID token
+of level decided by the user and fields `lat`, `lon`.
+
+The `_cid` tags hold value of cell ID expressed as token (`s2.CellID.ToToken()`) of corresponding level.
```

Comment:
I think it would be good to use `_s2cell` instead of `_ci` just so the meaning is really clear.  Likewise, maybe `_tid` should be `_track`?

Reply:
Telegraf team is recommending that we do NOT have "_"-based field names.  I believe we need to specify the tag name which represents the geohash in the function.  We could default to a tag value -- that we should make the default in the corresponding Telegraf plugin.  Similiarly, we should be able to default the lat/lon field names, but if the use wants to override this...we should allow it.  I believe there are other function signatures that do something similar?  Please suggest one to follow as an example.

Reply:
I'll double check, but I don't think that it's possible to handle dynamic field names for functions that are written in Flux.  We *can* do it for builtin functions that are written in Go though, but that doesn't help with the way this code is structured. There is an existing function to address this but it's a ways off: https://github.com/influxdata/flux/issues/1122

It might be possible to workaround this limitation with creative use of the `rename` function, but we don't have any examples of that.  How important is it to be able to configure the field names for `lat`, `lon`, `cellid` etc fields?

Reply:
`_ci` renamed to `s2cellID` and box/circle/polygon unified into single region parameter.